src/utils/scenarioParameters/factories/inputComponentsFactories/BasicNumberInputFactory.js

```diff
@@ -38,12 +38,17 @@ const create = (t, parameterData, parametersState, setParametersState, editMode)
     });
   }
 
+  let value = parametersState[parameterData.id];
+  if (value === undefined || value === null) {
```

Comment:
Sorry, I have been a little too quicly for the review and after watching it closer, I noticed that :
 
 (value === undefined || value === null)  is the same thing than (value == null)

Reply:
Replaced the check by `value == null` :+1: